src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -191,8 +191,8 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
 
   /** Called when this component is mounted. */
   mounted (): void {
-    // set or initialize authorization object
-    this.authorization = this.getContinuationAuthorization || { files: [] } as ContinuationAuthorizationIF
+    this.authorization = this.getContinuationAuthorization ||
+      { files: [], date: null, expiryDate: null } as ContinuationAuthorizationIF
```

Comment:
So, initially setting them to null fixed the reactivity issue?

Reply:
You got it.

Reply:
Remember, undefined local variables, and undefined object properties, are not reactive.

That's why, elsewhere, when I added properties to an object, I used `$set(object, property)`.

Reply:
Keeping this in mind.